planetarium/src/test/java/com/revature/steps/PlanetandMoonVisibilitySteps.java

```diff
@@ -1,10 +1,87 @@
 package com.revature.steps;
 
 import io.cucumber.java.en.Given;
+import com.revature.TestRunner;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import io.cucumber.java.en.And;
+import org.junit.Assert;
 
 public class PlanetandMoonVisibilitySteps {
     @Given("The user is on the login page {string}")
-    public void theUserIsOnTheLoginPage(String arg0) {
+    public void theUserIsOnTheLoginPage(String Host) {
+        TestRunner.driver.get(Host);
+    }
+
+    @When("The user enters the username {string}")
```

Comment:
Would it be easier to remove the enter username and password steps since these can be part of shared steps?